test/suite/test_backup03.py

```diff
@@ -95,8 +95,7 @@ def populate(self):
     # Compare the original and backed-up files using the wt dump command.
     def compare(self, uri):
         self.runWt(['dump', uri], outfilename='orig')
-        # Pass -R flag as we need to run recovery on the backup dir before calling dump.
-        self.runWt(['-h', self.dir, '-R', 'dump', uri], outfilename='backup')
+        self.runWt(['-h', self.dir, 'dump', uri], outfilename='backup')
```

Comment:
Not related to this change, but I think it would be good to have unit tests (in the `test_util##.py` category) that tests using the `-r` flag correctly on some database, and using it with recovery or salvage and checking for the error message.

Reply:
Agreed, I'll create a follow-on ticket once we merge this change.